### ISSUE TITLE
Missing Requirements.txt and Shifting Back The Database to Sqlite 

### ISSUE THREAD

**Issue by anuragkumart16** [Ashitosh9922/Content-Management-Platform-#1]
There are multiple modules like widget-tewaks and django-filter which is being used and not mentioned in requirements.txt which creates frustration while running the server.

Also, The database should be SQLite not mySQL for development purposes.

Kindly assign me this issue and add hacktoberfest-accepted and hacktoberfest2024 tags. I will fix the above issues and also leave the database config untouched( commenting the code), so that when the development is completed it becomes easy for deployment.

Also while searching for Django repos in which I could Contribute, I have never found such a clean and well written code, feels like you are a very compassionate coder. Kudos to you code.

Regards 
Anurag

**Comment by anuragkumart16** [comment]
@Ashitosh9922 I would Like to Connect to you on LinkedIn, Kindly drop you linkedin Profile Link.

regards
Anurag

**Comment by Ashitosh9922** [comment]
yeah but in actual project i had used MySQL db instead of sqlite. You can use sqllite as well as postgresql also.

**Comment by anuragkumart16** [comment]
okay, that works ,please assign this issue to me adding hacttoberfest-accepted tag and accept my merge request
